src/ActionMenu.tsx

```diff
@@ -0,0 +1,76 @@
+import {List, ListPropsBase, GroupedListProps} from './ActionList/List'
+import {Item, ItemProps} from './ActionList/Item'
+import {Divider} from './ActionList/Divider'
+import Button, {ButtonProps} from './Button'
+import React, {useCallback, useRef, useState} from 'react'
+import Overlay from './Overlay'
+import getRandomValues from 'get-random-values'
+
+export interface ActionMenuProps extends ListPropsBase, GroupedListProps {
```

Comment:
Need to sync with someone on how to base ActionMenuProps on ActionList props - this is giving a typescript error, as is extending ListProps.